lib/components/FigureCard/FigureCard.tsx

```diff
@@ -0,0 +1,16 @@
+export interface FigureCardProps {
+  /** Content shown on the card */
+  content: string | number;
+  /** Caption shown on the card */
+  caption?: string;
+}
+
+/**
+ * This is a card component for displaying a text figure with a caption.
+ */
+export const FigureCard = ({ content, caption }: FigureCardProps) => (
+  <figure className="flex w-min flex-col items-center rounded-[10px] bg-jod-light px-[40px] py-[15px] font-bold">
```

Comment:
If `caption` is given only it will be now read with the screenreader. If `caption` is not given, then `content` is read.

So one possible way to have them both read might be to figure have `aria-label` as a prop or assume something like:
```
aria-label={`${caption} ${content}`}
```